src/Base64.tsx

```diff
@@ -44,6 +44,19 @@ function Base64(): JSX.Element {
         setSeparatedLines(newSeparatedLines)
     }
 
+    function getBase64Text(): string {
+        return base64Encode(textInputValue, encodingName)
+    }
+
+    function copyBase64ToClipboard() {
+        navigator
+            .clipboard
+            .writeText(getBase64Text())
```

Comment:
Why not use the same method as the output textarea? Right now the clipboard might not contain what I see in the textarea. I would expect to copy what I see.

Reply:
That was a mistake when rebasing onto newer main.